api/Makefile

```
@@ -3,10 +3,15 @@
 SHELL := /bin/bash
 
 # add yarn CLI dev deps to PATH (for cross platform POSIX commands via shx)
-PATH := $(shell cd .. && yarn bin):$(PATH)
+# and also make an explicit version for shx for use in the shell function,
+# where PATH won’t be propagated
+yarn_path := $(shell cd .. && yarn bin)
+PATH := ${yarn_path}:${PATH}
+# Make an explicit version of this for shx so it can be used in the $(shell)
+# function, where the PATH won’t be propagated
+SHX := ${yarn_path}/shx
```

This forward slash appears to be working on Windows 10, but makes me a little nervous. For `node_modules/.bin` dependencies in `$(shell ...)` this also works:

```suggestion
SHX := npx shx
```